Content.Server/Objectives/ObjectivesSystem.cs

```diff
@@ -27,15 +29,20 @@ public sealed class ObjectivesSystem : SharedObjectivesSystem
     [Dependency] private readonly IRobustRandom _random = default!;
     [Dependency] private readonly EmergencyShuttleSystem _emergencyShuttle = default!;
     [Dependency] private readonly SharedJobSystem _job = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
 
     private IEnumerable<string>? _objectives;
 
+    public bool ShowGreentext { get; set; }
```

Comment:
```suggestion
    private bool _showGreentext;
```
can be private and does not need a getter and setter